price/priceService.ts

```diff
@@ -0,0 +1,21 @@
+const baseUrl:string = 'https://dydx-testnet.imperator.co/v4';
+
+function fetchCandles(pair:string,timeFrame:string):void{
+    fetch(`${baseUrl}/candles/perpetualMarkets/${pair}?resolution=${timeFrame}`)
+    .then(response => {
+        if(!response.ok){
+            throw new Error('Network response was not ok');
```

Comment:
You need to pass error data to the thrown error. In other words, there should/may be a filed in `response` that gives you more info about why the error happened.